blaze-client/src/main/scala/org/http4s/client/blaze/Http1Client.scala

```diff
@@ -25,7 +26,7 @@ object Http1Client {
   private def resource[F[_]](config: BlazeClientConfig)(implicit
       F: ConcurrentEffect[F]): Resource[F, Client[F]] = {
     val http1: ConnectionBuilder[F, BlazeConnection[F]] = new Http1Support(
-      sslContextOption = config.sslContext,
+      sslContextOption = config.sslContext.fold[SSLContextOption](SSLContextOption.NoSSL)(SSLContextOption.Provided),
```

Comment:
Hi @rossabaker - please take a look at this one per my commit's message. Thanks

Reply:
That looks right.  It's the same fold you'd use in preserving the old `withSslContextOption` signature.